src/schemas/40-10007/schema.js

```diff
@@ -310,6 +315,9 @@ let schema = {
             properties: {
               attachmentId: {
                 type: 'string'
+              },
```

Comment:
can we please add name to the required list?

Reply:
Done